### ISSUE TITLE
some enums have been changed to camelCase from PascalCase

### ISSUE THREAD

**Issue by p7r0x7** [00JCIV00/cova#65]
cova's build.zig is no longer valid.
```
/Users/p7r0x7/.cache/zig/p/122073922013698dd2c2339cb9fa37436408eb97d033f166b817d8a12193c47c2af3/build.zig:182:51: error: no field named 'Struct' in union 'builtin.Type'
    inline for (@typeInfo(generate.MetaDocConfig).Struct.fields) |field| {
                                                  ^~~~~~
/Users/p7r0x7/zig/lib/std/builtin.zig:257:18: note: union declared here
pub const Type = union(enum) {
                 ^~~~~
referenced by:
    build: /Users/p7r0x7/.cache/zig/p/122073922013698dd2c2339cb9fa37436408eb97d033f166b817d8a12193c47c2af3/build.zig:78:45
    runBuild__anon_66109: /Users/p7r0x7/zig/lib/std/Build.zig:2167:33
    9 reference(s) hidden; use '-freference-trace=11' to see all references
```

**Comment by 00JCIV00** [comment]
Appreciate you pointing this out.

The issue is much deeper than that unfortunately. I'll be doing a migration to v0.14 at some point, but I have several projects pinned to v0.13, so I want to make sure there's not any other issues I need to resolve for those first.

**Comment by p7r0x7** [comment]
Yeah I assumed you knew it was deeper; haven't been able to compile my stuff for over a month, though pinning to 13 is reasonable.

**Comment by 00JCIV00** [comment]
Yea, hopefully not too much longer.

I've considered using `builtin.version` to support both, but it's used in too many areas to support a bandaid fix like that.

**Comment by Ratakor** [comment]
You could follow zig master on main and have a tagged release for 0.13.0. This way both master and 0.13.0 users could use cova.

**Comment by p7r0x7** [comment]
@Ratakor  in the short term, https://github.com/Beyley/cova is also available.

**Comment by 00JCIV00** [comment]
> You could follow zig master on main and have a tagged release for 0.13.0. This way both master and 0.13.0 users could use cova.

Appreciate you showing interest in Cova!

Normally, I would do this, however I have several projects that I've tagged to Zig v0.13.0 because they require a level of stability. As I work on these projects I also push Cova forward with features and fixes needed, so Cova main needs to be on v0.13.0 for that. 

However, my plan is to do something similar soon. I'll add a branch for `zig_nightly` that'll effectively be the last tagged Cova release, but updated for the latest Zig. This branch won't see new features, but I'll keep it working for nightly as well as I can.
